egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -818,6 +818,14 @@ def __init__(self,
                                  the final model combination stage.  These
                                  models will themselves be averages of
                                  iteration-number ranges""")
+        self.parser.add_argument("--trainer.optimization.do-final-combination",
+                                 dest='do_final_combination', type=str,
+                                 action=common_lib.StrToBoolAction,
+                                 choices=["true", "false"], default=True,
+                                 help="""Use this to specify conducting final
```

Comment:
change the help string to:
`Set this to false to disable the final 'combine' stage (in this case we just use the last-numbered model as the final.mdl.`